.murdock

```diff
@@ -3,7 +3,11 @@
 : ${TEST_BOARDS_AVAILABLE:="esp32-wroom-32 samr21-xpro"}
 : ${TEST_BOARDS_LLVM_COMPILE:="iotlab-m3 native nrf52dk mulle nucleo-f401re samr21-xpro slstk3402a"}
 # we can't use '-' in the variable names, convert them to '_' to add new boards
-: ${TEST_KCONFIG_samr21_xpro:="examples/hello-world tests/periph_*"}
+: ${TEST_KCONFIG_samr21_xpro:="examples/hello-world tests/periph_*
+tests/driver_ad7746 tests/driver_adcxx1c tests/driver_ads101x tests/driver_adt101x
+tests/driver_adt7310 tests/driver_adxl345 tests/driver_aip31068 tests/driver_apa102
+tests/driver_apds99xx tests/driver_apds99xx_full tests/driver_at tests/driver_at24cxxx
+tests/driver_at24mac tests/driver_at25xxx tests/driver_at30tse75x tests/driver_ata8520e"}
```

Comment:
Why not `tests/driver_a*`?

Reply:
That was the intention at first, but could not get around all the needed things that at86rf215 needs in the test application without bloating this. 

> Some network devices are not in this PR given that many test applications bring the GNRC stack, and that's not yet modelled.

Reply:
Ahh Ok.